### PR TITLE
Minor improvement to Chinese expression

### PR DESCRIPTION
**Pull request by HtwoO**
I suggest you search the Chinese essay 论中文的常态与变态 by 余光中 and improve some of the expression in the Chinese documentation.

I'm writing this in English because I see the commit history are almost all in English.

### CODE REVIEW

**Comment by xen0n** [comment]
I personally object to doing this, because technical Chinese writing mostly does NOT benefit from the *typical Chinese expression characteristics* which are mostly implicit semantics that have to be inferred.

**Comment by xen0n** [comment]
Also see the natlang-style-guide doc in the website -- it is explicitly recommended to write in 欧化中文 whenever it's more appropriate to do so -- unambiguity is strictly more valuable to "language beauty" or "naturalness".

**Comment by xry111** [comment]
But for this specific case it seems improving the beauty won't affect unambiguity, so why not?

FWIW in some cases natural Chinese is more unambiguity than English.  For example in the LFS book there are some "the test named `test-something` is known to fail" but I translate these phrases differently to "已知名为 `test-something` 的测试**会**失败" and "已知名为 `test-something` 的测试**可能**失败."

I do this because I'm a LFS editor and I'm pretty sure how to analysis them case by case.  But in English they are just the same, and in the mail list there are occasionally complains saying "hey, you say `test-something` is known to fail but it does not fail for me."

**Comment by xen0n** [comment]
> But for this specific case it seems improving the beauty won't affect unambiguity, so why not?

Let me explain more in another review... yesterday I was feeling a bit sick so I could only review on my phone.

> FWIW in some cases natural Chinese is more unambiguous than English. For example in the LFS book there are some "the test named `test-something` is known to fail" but I translate these phrases differently to "已知名为 `test-something` 的测试**会**失败" and "已知名为 `test-something` 的测试**可能**失败."
> 
> I do this because I'm a LFS editor and I'm pretty sure how to analysis them case by case. But in English they are just the same, and in the mail list there are occasionally complains saying "hey, you say `test-something` is known to fail but it does not fail for me."

For the latter case, no they can be expressed differently: "it's known that the test case ... can fail" vs "it's known that the test case ... fails"; or maybe "the test case named ... is known to fail *sometimes*".

**Review by xen0n — COMMENTED** [review]
Thanks for your contribution (even though we may disagree a lot about Chinese language usage ~~or each other's true nationality~~).

Anyway, even if we're to style this *technical* website the *literary* or 余光中 way, there are far more places that needs editing: first of all the recommendation to use 欧化中文 must be reworded, then one would have to go through every single doc to remove 被 markers inferrable from context (e.g. animacy). If expressions like "一般被认为……的用户" or even "描述式" are considered eyesore, then something like "这些被捆绑的旧世界动态链接库，会先于新世界宿主系统提供的动态链接库，被搜索和加载。" would certainly be something burning from hell...

And, as you're touching *the natlang-style-guide doc itself* (which I didn't notice yesterday), I'd like to reiterate what I want to *experiment* with the particular writing style: to minimize *possibility* of confusion, and to *objectively* do that with explicit recommendation for gratuitous usage of grammatical markers / otherwise redundant information. I have seen and written technical docs in my previous job both in this way and not, and the team's experience is the "more natural / oral" versions require lots of back-and-forth communication to have ambiguities resolved, which is almost always the case; while the 欧化中文 docs by me mostly can be understood by anyone else without having to consult me.

So, in this patch's current shape, I'm likely to accept the last change with modification (changing your "具体的" to "生动的" which better matches the "deep structure" of my thoughts), but decline the other two. Thanks for sharing your thoughts again.

**Comment by xen0n** [comment]
BTW, please think about the example I gave in the doc -- 「`AM*` 原子访存指令如果 `rd` 和 `rj` 的寄存器号相同，则触发指令不存在例外。」 where everyone already familiar with Loongson didn't have a problem comprehending it, but everyone reading about LoongArch for the first time had parsed the syntax in the wrong way. Sure, the LoongArch manual only mentioned 指令不存在例外 *after* the instructions (and this sentence), but you can see that some *classes* of ambiguities is impossible for experienced contributors to notice with conventional oral speech, so the extra information provided by 欧化中文 serves to objectively improve comprehension for everyone alike.

File: docs/contrib/natlang-style-guide.md [hunk 1]
```diff
@@ -158,7 +157,7 @@ sidebar_position: 2
 
 不要使用「点击这里怎么怎么样」或类似的表达。
 例如，不要写「点击<a>这里</a>查看活动详细信息」，
-而用「活动详情请见<a>主办方页面</a>」「活动主办方也<a>设置了</a>详细信息页面」等更加描述式的写法。
+而用「活动详情请见<a>主办方页面</a>」「活动主办方也<a>设置了</a>详细信息页面」等更加具体的描述。
```
[the remark this answers is not in the document]
Comment (HtwoO) [reply docs/contrib/natlang-style-guide.md:160]: Initially I only wanted to change this as it's obviously from "descriptive" in English, but a direct "translation" to Chinese is rather odd for Chinese readers. And I somehow think this alone is too small a change. Then I saw no glossary for "i18n" after `grep`ing in the project, so I made a few more changes.

**Comment by HtwoO** [comment]
I understand that it is important to be accurate in technical writing. I personally also prefer unambiguity over beauty and naturalness when writing documentation. But in many cases we can write unambiguously and naturally.

The purpose of a documentation is to help readers understand the subject matter. In other words, the goal of an article is to communicate (although in many cases it's just one way, namely, authors -> readers). Thus many organizations now encourage people to use plain words in writing/speaking.

Writing naturally and concisely usually help readers understand the content better. Ideally technical documents should include a "prerequisites" paragraph to provide assumptions for further content for easier reproducibility at audience side.

reference:
https://www.justice.gov/open/plain-writing-act
https://en.wikipedia.org/wiki/Corporate_jargon#Criticism
https://en.wikipedia.org/wiki/Wikipedia:Use_plain_English.

File: docs/contrib/natlang-style-guide.md [hunk 2]
```diff
@@ -7,12 +7,11 @@ sidebar_position: 2
 :::warning 施工现场！
 本文目前尚不完整。维护者想起来更新或被催更，则会更新；否则请不要认为所有相关的规范都完整记录了。
 
-由于本站的 i18n 工作仍未完成，本文目前只覆盖汉语文本。
-后续适用于英语文本的内容仍待记录。
+由于本站的 i18n （internationalization，即「国际化」）工作仍未完成，本文目前只有中文版本，后续仍需补充英文内容。
```
[the remark this answers is not in the document]
Comment (HtwoO) [reply docs/contrib/natlang-style-guide.md:10]: Whether or not `本文` includes the _future_ English version depends on the interpreter. Maybe `本项目` is better.

File: docs/contrib/natlang-style-guide.md [hunk 3]
```diff
@@ -7,12 +7,11 @@ sidebar_position: 2
...
 :::
 
-随着龙架构的生态成熟、市场份额扩张，先前一般被认为不会接触此技术的用户也纷至沓来。
-这一般是好事，但不巧的是：
+随着龙架构的生态成熟、市场份额扩张，先前以为不会接触此技术的用户也纷至沓来。
+这本来是好事，但不巧的是：
```
[the remark this answers is not in the document]
Comment (xry111) [reply docs/contrib/natlang-style-guide.md:14]: I'd say this change is wrong.  Personally I don't think "some U.S. government agency has started to hack on LoongArch" a "good" thing.  Maybe the others don't agree but "good" or not is highly subjective anyway.

File: docs/contrib/natlang-style-guide.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (xry111) [reply docs/contrib/natlang-style-guide.md:10]: “本文目前只描述汉语文本的语言惯例”。

应该是这个意思。

**Comment by xry111** [comment]
> BTW, please think about the example I gave in the doc -- 「`AM*` 原子访存指令如果 `rd` 和 `rj` 的寄存器号相同，则触发指令不存在例外。」 where everyone already familiar with Loongson didn't have a problem comprehending it, but everyone reading about LoongArch for the first time had parsed the syntax in the wrong way. Sure, the LoongArch manual only mentioned 指令不存在例外 _after_ the instructions (and this sentence), but you can see that some _classes_ of ambiguities is impossible for experienced contributors to notice with conventional oral speech, so the extra information provided by 欧化中文 serves to objectively improve comprehension for everyone alike.

This sentence is certainly ambiguous even only considering the "normal" Chinese writing.  And even the Loongson translator (I can reasonably assume he/she is familiar with LoongArch) could not understand this sentence so the initial version of the English manual was wrong here.

**Pull request closed without merging**